sys/arm64/arm64/support.S

```diff
@@ -145,8 +182,22 @@ EENTRY(fueword64)
 EEND(fueword64)
 END(fueword)
 
+#if __has_feature(capabilities)
+/*
+ * int fuecap(volatile const void * __capability, intcap_t *)
+ */
+ENTRY(fuecap)
+	check_user_access 0, (VM_MAXUSER_ADDRESS-15), fsu_fault_nopcb
+	fsudata	ldtr, c0, 0
+	str	c0, [x1]		/* Save the data in kernel space */
+	mov	x0, #0			/* Success */
+	ret				/* Return */
+END(fuecap)
+#endif
+
 /*
- * int subyte(volatile void *, int)
+ * int subyte(volatile void * __capability, int)
+ * TODO: __capability
```

Comment:
Are these really still TODO given the common macro in use?

Reply:
They are left over from when I was reworking the code & I missed them